apps/api/src/donations/donations.service.ts

```diff
@@ -142,8 +142,11 @@ export class DonationsService {
 
   async remove(ids: string[]): Promise<DeleteManyResponse> {
     try {
-      return await this.prisma.donation.deleteMany({
+      return await this.prisma.donation.updateMany({
         where: { id: { in: ids } },
+        data: {
+          status: 'deleted',
```

Comment:
Please use the status from `DonationStatus` enum

Reply:
Thank you, I fixed it.